packages/strapi-design-system/src/DateTimePicker/DateTimePickerProps.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const DateTimePickerProps = (props) => <div {...props} />;
+
+export const dateTimePickerDefaultProps = {
+  ariaLabel: undefined,
+  clearLabel: 'clear',
+  disabled: false,
+  error: undefined,
+  hint: undefined,
+  label: undefined,
+  labelAction: undefined,
+  onClear: undefined,
+  required: false,
+  size: 'M',
+  step: 1,
+  value: undefined,
+};
+export const dateTimePickerPropTypes = {
+  /**
+   * Aria Label used by the DatePicker and the TimePicker inside the DateTimePicker component
+   */
+  ariaLabel: PropTypes.string,
+  /**
+   * Clear Label used for the X button inside both, the TimePicker and the DatePicker
+   */
+  clearLabel: PropTypes.string,
+  /**
+   * Disabled attribute
+   */
+  disabled: PropTypes.bool,
+  /**
+   * Error message or boolean, if it is a boolean we will hide the error message for the DatePicker and the TimePicker and show just the one that we have defined in the DateTimePicker
```

Comment:
basically the error message inside the DateTimePicker is always a boolean, I need to change the comment...It is a boolean because we want to hide the Error messages for the DatePicker and TimePicker

Reply:
Sorry, I confused the errors, I can wrote a better comment